MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1152 @@
+</td>
+</tr>
+<tr>
+<td>
+Checkbox
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[CheckBox](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/checkbox?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Switch
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+{{'[Switch](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/switch?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+Picker
+</td>
+<td>
+Enum and List type property. 
+</td>
+<td>
+{{'[Picker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+ComboBox
+</td>
+<td>
+Enum and List type property.
+</td>
+<td>
+{{'[SfComboBox](https://help.syncfusion.com/maui/combobox)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+AutoComplete
+</td>
+<td>
+Enum and List type property.
+</td>
+<td>
+{{'[SfAutoComplete](https://help.syncfusion.com/maui/autocomplete)'| markdownify }}
+</td>
+</tr>
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+Enum and List type property.
+</td>
+<td>
+{{'[RadioButton](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/radiobutton?view=net-maui-7.0)'| markdownify }}
+</td>
+</tr>
+</table>
+
+## To change the editor for any data type
+
+By default, the editors will be loaded based on the datatype. To change the editor for any type, use the `RegisterEditor` method and specify the type and editor.
+
+{% tabs %}
+{% highlight C# %}
+
+    this.dataForm.RegisterEditor(typeof(string), DataFormEditorType.MultilineText);
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the `MultilineText` editor will be loaded for the string type instead of `Text` editor.
+
+## Changing the editor for property
+
+To change the editor for any property, use the `RegisterEditor` method and specify the property name and editor.
+
+{% tabs %}
+{% highlight c# %}
+
+    this.dataForm.RegisterEditor("On", DataFormEditorType.Switch);
+
+{% endhighlight %}
+{% endtabs %}
+
+Here, the Switch editor will be loaded for the `On` property (bool type) instead of `CheckBox` editor.
+
+## Text editor
+
+In the text editor, the [Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0) is loaded.
+
+Text editor have Keyboard type change by using `Keyboard` property and you can set maximum allowed length of input by using `MaxLength` property.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.Text)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Multiline Text editor
+
+In the `MultilineText` editor, the [Editor](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-7.0) is loaded.
+
+And `MultilineText` editor height will auto expand/reduce based on the line wraps in editor , which allowing text to be readable without scrolling the editor.
```

Comment:
`MultilineText` editor height will auto expand or reduce based on the line wraps, allowing text to be readable without scrolling the editor.

Reply:
content updated